schema/experiment.yml

```diff
@@ -87,10 +87,11 @@ $defs:
   ControlVariant:
     type: object
     additionalProperties: false
+    description: A variant used as an experimental control to measure assay concorance
     properties:
-      vrs:
+      allele:
         description: GA4GH VRS object describing the variant
-        $ref: "https://github.com/ga4gh/vrs/blob/main/schema/vrs.json#definitions/Allele"
+        $ref: "https://raw.githubusercontent.com/ga4gh/vrs/2.x/schema/vrs.json#definitions/Allele"
```

Comment:
This is the only technical issue I found. You will want to reference the raw JSON file so this is readable by schema processing tools. I also linked this to the `2.x` version, but you should update this to a specific release before implementing.